src/include/binder/copy/bound_copy_to.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "binder/bound_statement.h"
+#include "binder/query/bound_regular_query.h"
+#include "catalog/table_schema.h"
```

Comment:
doesn't seem to be needed

Reply:
That's right, only `bound_regular_query.h` is needed. I'm removing the others